src/Microsoft.AspNet.Mvc.ViewFeatures/Controller.cs

```diff
@@ -508,6 +508,12 @@ public virtual ContentResult Content(string content, MediaTypeHeaderValue conten
             return result;
         }
 
+        [NonAction]
+        public virtual NoContentResult NoContent()
```

Comment:
Yeah feel free to steal from MSDN from Web API or MVC or wherever we might have had a similar API before.